packages/core/saved-objects/core-saved-objects-common/src/saved_objects_error_helpers.ts

```diff
@@ -0,0 +1,448 @@
+/*
```

Comment:
I moved the error helpers definitions to common to resolve a circular reference (which resulted from moving the authz logic from packages to x-pack/plugins/...). I may be possible to move somewhere else (like core-saved-objects-server), but I chose common. Let me know if there is a more suitable package.